caddy/setup/browse.go

```diff
@@ -112,12 +112,22 @@ header,
 
 th:first-child,
 td:first-child {
-	padding-left: 5%;
+	padding: 0.5em 2.5% 0.5em 5%;
+}
+
+td:first-child svg {
+	position: absolute;
+}
+
+td:first-child div.name {
+	margin-left: 2em;
+	word-break: break-all;
+	overflow-wrap: break-word;
```

Comment:
Ah yes, these are good changes (along with the vertical-align: top a few lines below) - I've incorporated those locally and like them a lot.